src/components/UploadingJobsContextProvider/UploadingJobsContextProvider.js

```diff
@@ -55,11 +54,35 @@ class UploadingJobsContextProviderComponent extends Component {
   updateUploadDefinition = async newUploadDefinition => {
     const uploadDefinition = newUploadDefinition || await this.getLatestUploadDefinition();
 
+    if (this.isUploadDefinitionFailed(uploadDefinition)) {
+      return this.deleteUploadDefinition();
+    }
+
     this.setState({ uploadDefinition });
 
     return uploadDefinition;
   };
 
+  isUploadDefinitionFailed(uploadDefinition) {
+    if (isEmpty(uploadDefinition)) {
+      return false;
+    }
+
+    const {
+      status: uploadDefinitionStatus,
+      fileDefinitions,
+    } = uploadDefinition;
+    const isErrorStatus = uploadDefinitionStatus === FILE_STATUSES.ERROR;
+    const isAllFilesFailed = every(
+      fileDefinitions,
+      ({ status }) => {
```

Comment:
`({ status }) => status === FILE_STATUSES.ERROR`

Moreover, let's be consistent and put
`file => file.status === FILE_STATUSES.ERROR`
like you did
https://github.com/folio-org/ui-data-import/pull/134/files#diff-a5d08c98e653a55d85a505fd08109c36R157